Cargo.toml

```diff
@@ -102,7 +102,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [[example]]
 name = "smtp"
-required-features = ["smtp-transport"]
+required-features = ["smtp-transport", "builder"]
```

Comment:
It's strange that this needs to be set for the smtp example but not for the rest of them, is this still necessary?